arch/sim/src/sim/up_hostirq.c

```diff
@@ -86,3 +112,44 @@ void up_irq_restore(uint64_t flags)
 void up_irqinitialize(void)
 {
 }
+
+/****************************************************************************
+ * Name: up_enable_irq
+ *
+ * Description:
+ *   Enable the IRQ specified by 'irq'
+ *
+ ****************************************************************************/
+
+void up_enable_irq(int irq)
```

Comment:
Q: This function will enable interrupts for a CPU. The behaviour is that child threads will copy the signal mask upon creation. Do you think it's better for us to block all the signals in `sim_idle_trampoline` except the `SIGUSR1` ?

Reply:
It's good to unmask the signal on all threads because we can simulate the hardware behaviour more closely.